app/js/app/controllers/MyBoardsPageControllers.js

```diff
@@ -28,9 +28,15 @@ define([], function() {
 		$scope.filterOption = $scope.filterOptions[0];
 		$scope.sortOption = $scope.sortOptions[1];
 
+		var roundUpToNearestSix = function(initialValue) {
+			var valueModuloSix = initialValue % 6;
+			return valueModuloSix || initialValue == 0 ? initialValue + 6 - valueModuloSix : initialValue;
```

Comment:
This confused me! I thought it would work out the truthy-ness of `valueModuloSix` and if it was false-y return `initialValue == 0 ? initialValue + 6 - valueModuloSix : initialValue`. Apparently it doesn't do this!